### PR TITLE
docs(readme): Foo integration

### PR DESCRIPTION
**Pull request by adamhenson**
**Summary**

> What kind of change does this PR introduce?

Adds an integration: [Foo](https://www.foo.software).

Thanks in advance 🙇

### CODE REVIEW

**Comment by adamhenson** [comment]
I signed it!

File: readme.md [hunk 1]
```diff
@@ -280,6 +280,8 @@ This section details services that have integrated Lighthouse data. If you're wo
 
 * **[Fluxguard](https://fluxguard.com/)** - Fluxguard provides website DOM change monitoring orchestrated with Google Puppeteer, and audited by Lighthouse. Fluxguard is a freemium product, with monthly monitoring of up to 75 pages for free.
 
+* **[Foo](https://www.foo.software)** - Foo continuously captures performance audits with Lighthouse, notifies of change, and provides visualization. Users can choose to be notified via email, Slack and PagerDuty. A dashboard with charts show changes by day, week or month and provide detailed reports.
```
Comment (patrickhulce) [inline readme.md]: Could you add the note that this is a paid product? It also isn't clear if there is any sort of free trial.

Comment (adamhenson) [reply readme.md]: Thanks for reviewing @patrickhulce. Updated in 9065b24. If you had different phrasing in mind - feel free to let me know.

File: readme.md [hunk 2]
```diff
@@ -280,6 +280,8 @@ This section details services that have integrated Lighthouse data. If you're wo
 
 * **[Fluxguard](https://fluxguard.com/)** - Fluxguard provides website DOM change monitoring orchestrated with Google Puppeteer, and audited by Lighthouse. Fluxguard is a freemium product, with monthly monitoring of up to 75 pages for free.
 
+* **[Foo](https://www.foo.software)** - Foo continuously captures performance audits with Lighthouse, notifies of change, and provides visualization. Users can choose to be notified via email, Slack and PagerDuty. A dashboard with charts show changes by day, week or month and provide detailed reports. Foo is a paid service available with tiers of pricing. Free trials aren't offered at this time, but a [demo dashboard](https://www.foo.software/demo/dashboard) is publicly available for viewing.
```
Comment (patrickhulce) [inline readme.md]:
```suggestion
* **[Foo](https://www.foo.software)** - Foo continuously monitors performance with Lighthouse, monitors for changes, and provides detailed visualization. Users can receive alerts via email, Slack, and PagerDuty with daily, weekly, or monthly reports. Foo is a paid service with several tiers of pricing.
```

Comment (patrickhulce) [reply readme.md]: offered a few suggestions to trim it down to the average size here, but feel free to tweak :)

Comment (adamhenson) [reply readme.md]: Thanks @patrickhulce - I took your suggestion and trimmed it even further (removed duplicate `monitors` wording). Ready for re-review when you all have time. Thanks again.

**Comment by adamhenson** [comment]
Thanks for the quick review @patrickhulce and @exterkamp.

**Comment by adamhenson** [comment]
@patrickhulce - please let me know if there’s anything else I can do to have this merged. Thanks again for your time!

**Comment by adamhenson** [comment]
Thanks for the quick turnaround everyone. Keep up the good work on this awesome project 💗

**Pull request merged**